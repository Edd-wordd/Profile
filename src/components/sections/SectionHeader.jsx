import React from 'react'
import { Hidden, Typography, Grid } from '@material-ui/core'
import { useStyles } from '../styles/sections/SectionHeader.styles'

function SectionHeader(props) {
  const classes = useStyles(props)
  return (
    <>
      <Grid className={classes.titleWrapper} container direction="row" justify="center">
        <Typography variant="h4" className={classes.title}>
          {props.title}
        </Typography>
        <Grid container item direction="row" justify="center">
          <Hidden xsDown>
            <div className={classes.lineL}></div>
            <div className={classes.diamond}></div>
            <div className={classes.lineR}></div>
          </Hidden>
          <Hidden smUp>
            <div className={classes.lineL}></div>
            <div className={classes.diamond}></div>
            <div className={classes.lineR}></div>
          </Hidden>
        </Grid>
        <Typography className={classes.subTitle} variant="h6">
          {props.subTitle}
        </Typography>
      </Grid>
    </>
  )
}

export default SectionHeader
