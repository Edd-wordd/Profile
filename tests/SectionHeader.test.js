import React from 'react'
import testComponent from './testComponent'
import SectionHeader from '../src/components/sections/SectionHeader'
testComponent(<SectionHeader />, 'SectionHeader')
