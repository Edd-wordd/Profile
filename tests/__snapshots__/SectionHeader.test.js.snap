// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resources should render Resources 1`] = `"Component"`;

exports[`SectionHeader should render SectionHeader 1`] = `"Component"`;
